dioptas/widgets/plot_widgets/HistogramLUTItem.py

```diff
@@ -321,10 +328,14 @@ def _configurationButtonClicked(self):
         if isinstance(self.imageItem, NormalizedImageItem):
             widget.setCurrentNormalization(self.imageItem.getNormalization())
         widget.setRange(*self.getExpLevels())
-        widget.setDataHistogram(counts=self.hist_y, bins=self.hist_x)
+        widget.setData(data=self.getImageData(copy=False), copy=False)
         widget.sigCurrentGradientChanged.connect(self._configurationGradientChanged)
         widget.sigCurrentNormalizationChanged.connect(self._normalizationChanged)
-        widget.sigRangeChanged.connect(self.setLevels)
+        def rangeChanged(minimum, maximum):
+            self.setLevels(minimum, maximum)
+            # Update displayed range since it can differ from the provided one
+            widget.setRange(*self.getExpLevels())
+        widget.sigRangeChanged.connect(rangeChanged)
```

Comment:
This makes sure that the diplayed range is the one used.
Beforehand, it was not taking into account the possible changes that `setLevels` does on the range (e.g., a negative min value gets changed)